app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
maybe you can delete this one and only keep the .gitignore from the root project.